core/trino-main/src/test/java/io/trino/sql/query/TestJsonQueryFunction.java

```diff
@@ -439,4 +439,38 @@ SELECT json_query(
                         VALUES cast('[[true,{"c":{"c":null}}],{"c":null},null]'AS varchar)
                         """);
     }
+
+    @Test
+    public void testDescendantMemberSingleChildWithoutArrayWrapperAccessor()
+    {
+        // Test json_query to read the result of a single descendant member in json
+        assertThat(assertions.query(
+                """
+                select json_query(
+                                '{"author":"Diana","id":{"value":1},"notes":[{"type":1,"comment":"foo"},{"type":2,"comment":null}],"comment":["bar","baz"]}',
+                                'lax $..author'
+                                omit quotes)
+                """))
+                .matches(
+                        """
+                        VALUES cast('Diana' as varchar)
+                    """);
+    }
+
+    @Test
+    public void testDescendantMemberMultipleChildrenWithoutArrayWrapperAccessor()
+    {
+        // Test json_query to read the result of a multiple descendant member in json
```

Comment:
we could add a simple description in comment for why hit more than one elements return null